arch/arm/soc/st_stm32/stm32l4/Kconfig.defconfig.stm32l433xx

```diff
@@ -0,0 +1,27 @@
+# Kconfig - ST Microelectronics STM32L433XX MCU
+#
+# Copyright (c) 2016 Open-RnD Sp. z o.o.
+# Copyright (c) 2016 BayLibre, SAS
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+if SOC_STM32L433XC
+
+config SOC
+	string
+	default "stm32l433xx"
+
+config NUM_IRQS
+	int
+	default 82
```

Comment:
83? The IRQ numbers start from 0.

Reply:
L433 has the same number of IRQs as the L432 (where I copied this from); the additional IRQs for extra peripherals are interspersed in places where the L432 has holes.

Now that I'm looking at it though, I'm not sure the number is correct for the L432 either; at least 82 is the CRS interrupt, which exists in the entire L43x series. Maybe it's not used(?)

Reply:
Then we need also to correct it on L432. I will create a separate PR for this.